rfcs/text/semantic-release.md

```diff
@@ -0,0 +1,249 @@
+- Start Date: 2019-11-28
+
+# Summary
+
+We, the Carbon team, want to move to a continuous delivery approach. We have identified [`semantic-release`](https://github.com/semantic-release/semantic-release) as a suitable
+choice.
+
+# Basic example
+
+https://semantic-release.gitbook.io/semantic-release/
+
+# Motivation
+
+The number of changes included between `v8` and `v9` were substantial. It has become difficult for users to upgrade due
+to the number of changes. This is due to not releasing fast enough for the throughput of the team.
+
+We want to release faster so users can receive features faster and make small incremental upgrades. This will make
+breaking changes less of an issue for our consumers.
+
+# Detailed design
+
+We will use [`semantic-release`](https://github.com/semantic-release/semantic-release) to publish our package.
+- we will release from `master`
+- every commit to `master` will trigger a release
```

Comment:
Does really `every` commit to `master` trigger a release? According to the table with release types shown below it seems that not all commits trigger a release.

Reply:
Good spot. I'll update that.